app/util/Raven.js

```diff
@@ -1,22 +1,23 @@
-import config from '../config';
 import { COMMIT_ID } from '../buildInfo';
 
-function getRaven() {
-  if (process.env.NODE_ENV === 'production') {
-    /* eslint-disable global-require */
-    const Raven = require('raven-js');
-    Raven.addPlugin(require('raven-js/plugins/console'));
-    /* eslint-enable global-require */
+function getRaven(config) {
+  return function createRaven() {
```

Comment:
Right - config is visible in the client.js where plugin is defined.  Wrapper is indeed redundant, removed.